canonical-web-and-design/jaas-dashboard
# Add apps panel to models details page

## Done

- Added apps panel to model details page

## QA

- Pull feature branch
- Go to models page, click a model
- On model details page, click an app
- Verify side panel opens from the right as expected with correct model info

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1368
